test/notebooks/test_notebooks.py

```diff
@@ -133,18 +120,3 @@ def test_mitigating_disparities_in_ranking_from_binary_data():
         "sel_expgrad_X_alt.loc[ 'disparity', :][0]",
         pytest.approx(0.35, abs=0.08))
     assay_one_notebook(nb_name, test_values)
-
-
-@pytest.mark.notebooks
-def test_binary_classification_with_the_uci_credit_card_default_dataset():
-    nb_name = "Binary Classification with the UCI Credit-card Default Dataset"
-    test_values = {}
-    test_values["GridSearch_17"] = ScrapSpec(
```

Comment:
I'm not seeing this test in the new `.py` file. Am I missing something? It's not a great test, but it does ensure that the notebook has run and produced results as expected.

Reply:
With notebooks moved out of the notebooks directory we don't have these tests anymore.

Reply:
We're losing that functionality? I assume that the notebooks at least get run (although I can't remember where that happens now).... can we have some function like this back?

Reply:
They get run because that's how the webpage is generated.

Reply:
Thanks. I'm not hugely keen on the loss of the assertions - without them, there's a bit of a risk of silent failures. However, I don't see an easy way to add them back in - you can't just run the Python files because they pop up things from `matplotlib`.